index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Proyect Capstone</title>
+</head>
+
+<body>
+    
```

Comment:
- [ ] fFrom the project requirements, the webapp should on Desktop should be  `768px` or wider. From your current design, the desktop view has an empty space by the left from the `768` specified. Kindly look into this and rectify 👍 

### View:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/5962fa9b-786f-4335-b74d-f44ee44908e6)

Reply:
Thank you very much for all the suggestions, the changes have been made satisfactorily 👍😊